Sources/MapboxNavigation/RouteLineController.swift

```diff
@@ -139,17 +139,17 @@ extension NavigationMapView {
         // MARK: - NavigationComponentDelegate implementation
         
         func navigationViewDidLoad(_ view: UIView) {
-            navigationMapView.mapView.mapboxMap.onNext(.styleLoaded) { [self] _ in
-                navigationMapView.localizeLabels()
-                navigationMapView.mapView.showsTraffic = false
+            navigationMapView.mapView.mapboxMap.onNext(.styleLoaded) { [weak self] _ in
+                self?.navigationMapView.localizeLabels()
+                self?.navigationMapView.mapView.showsTraffic = false
```

Comment:
In theory `self` can be deallocated between these lines, so as a rule of thumb I always add `guard let self = self else { return }` if self is used more than once.
Not sure if it's critical in this case.